src/Components/BranchesControl.jsx

```diff
@@ -76,7 +76,7 @@ export default function Branches() {
       {branches.length > 1 && modelPath.repo !== undefined &&
         <Paper elevation={0} variant='control'
           sx={{
-            marginTop: '34px',
```

Comment:
@Ibrahim5aad @OlegMoshkovich Is the change in margin due to how the page renders on different devices? Is this a one-off change or do we need breakpoints to handle different screen sizes?

Reply:
@oo-bldrs Nope, this just visual alignment I though it is convenient after I introduced another component.  It this exact value serves another purposes I'm not aware of I can revert this.

Reply:
See my overall comment about moving this into the NavTree.  Maybe the alignment will change.